projects/openapi-io/src/parser/sourcemap.ts

```diff
@@ -72,31 +82,3 @@ export class JsonSchemaSourcemap {
     }
   }
 }
-export function resolveJsonPointerInYamlAst(
```

Comment:
This is unused, it was split out previously into `sourcemap` and `sourcemapReader` (which this belongs to the latter). It's duplicated in sourcemapReader where it's actually used